### PR TITLE
Support env in UFM service

### PR DESCRIPTION
**Pull request by boyuan-chen**
@rupertbg As we discussed, we can use environment variables in the UFM service to simplify the deployment process and improve security.

### CODE REVIEW

File: op-ufm/pkg/config/config.go [hunk 1]
```diff
@@ -79,6 +80,27 @@ func New(file string) (*Config, error) {
 	if _, err := toml.DecodeFile(file, cfg); err != nil {
 		return nil, err
 	}
+
+	KMSKeyID := os.Getenv("KMS_KEY_ID")
+	KMSEndPoint := os.Getenv("KMS_ENDPOINT")
+	KMSRegion := os.Getenv("KMS_REGION")
+	for _, wallet := range cfg.Wallets {
```
Comment (jyellick) [inline op-ufm/pkg/config/config.go]: Seems like kind of odd behavior to set all configured wallets with the same kms info?  Maybe it's worth at least logging that an override is happening?

I haven't looked, but I'm assuming that the configuration isn't using something like urfavcli that has first class environment overrides?

Comment (boyuan-chen) [reply op-ufm/pkg/config/config.go]: There is no `urfavcli`. It loads the whole configuration from the toml file. It just simplifies @rupertbg 's work.

**Review by mmontour1306 — APPROVED** [review]

**Comment by boyuan-chen** [comment]
@rupertbg This should be enough for the deployment, right?

**Comment by boyuan-chen** [comment]
@InoMurko I moved the `os import` to a different file and solve the building issue.

**Review by InoMurko — COMMENTED** [review]
Baking the toml in means we will need to go through a release cycle just to change a parameter. That does not seem right to me.

**Comment by jyellick** [comment]
> Baking the toml in means we will need to go through a release cycle just to change a parameter. That does not seem right to me.

@InoMurko FWIW, I certainly agree in principle that we should be injecting the config file in at runtime but apparently we have troubles with this operationally.  To my understanding, without a config.toml file, there are no defaults for these configuration values, and, the values we need to change can be overridden with env.

To me, the best solution would be to have the binary adopt sane defaults in the absence of a config file, overriding only what we need to, but this would require a much more substantial diff.  So, unless we are interested in pushing such changes upstream, baking in some config defaults in the image didn't seem like such a bad alternative.

**Comment by InoMurko** [comment]
> > Baking the toml in means we will need to go through a release cycle just to change a parameter. That does not seem right to me.
> 
> 
> 
> @InoMurko FWIW, I certainly agree in principle that we should be injecting the config file in at runtime but apparently we have troubles with this operationally.  To my understanding, without a config.toml file, there are no defaults for these configuration values, and, the values we need to change can be overridden with env.
> 
> 
> 
> To me, the best solution would be to have the binary adopt sane defaults in the absence of a config file, overriding only what we need to, but this would require a much more substantial diff.  So, unless we are interested in pushing such changes upstream, baking in some config defaults in the image didn't seem like such a bad alternative.

Agreed makes sense.

**Review by InoMurko — APPROVED** [review]

**Pull request merged**